tests/helics/application_api/CMakeLists.txt

```diff
@@ -7,57 +7,91 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-set(application_api_test_headers testFixtures.hpp ValueFederateTestTemplates.hpp
-                                 ../coreTypeLists.hpp
-)
+set(common_test_headers testFixtures.hpp ../coreTypeLists.hpp)
 
-set(application_api_test_sources
-    MessageFederateKeyTests.cpp
+set(common_test_sources testFixtures.cpp applicationApiTests.cpp)
+
+set(valueFederate_test_sources
     ValueFederateKeyTests.cpp
-    FilterTests.cpp
     TranslatorTests.cpp
-    testFixtures.cpp
+    ValueFederateTestTemplates.hpp
     PrimaryTypeConversionTests.cpp
-    CallbackFederateTests.cpp
-    FilterAdditionalTests.cpp
     ValueConverterTests.cpp
-    application-api-tests.cpp
     data_viewTests.cpp
-    MessageFederateAdditionalTests.cpp
-    FederateTests.cpp
     subPubObjectTests.cpp
-    GrantTimeoutTests.cpp
     ValueFederateAdditionalTests.cpp
-    CombinationFederateTests.cpp
     ValueFederateSingleTransfer.cpp
     ValueFederateDualTransfer.cpp
     helicsTypeTests.cpp
+    MultiInputTests.cpp
+)
+
+set(messageFederate_test_sources MessageFederateKeyTests.cpp MessageFederateAdditionalTests.cpp
+                                 ValueFederateAdditionalTests.cpp CombinationFederateTests.cpp
```

Comment:
`ValueFederateAdditionalTests` doesn't seem like it should be part of the message federate tests.